plugins/gcp/resoto_plugin_gcp/resources/compute.py

```diff
@@ -767,6 +767,58 @@ class GcpDiskType(GcpResource, BaseVolumeType):
     default_disk_size_gb: Optional[str] = field(default=None)
     valid_disk_size: Optional[str] = field(default=None)
 
+    RESOURCE_GROUP_MAP = {
```

Comment:
This is defined in a data class - it becomes an attribute of every instance if not defined as classvar explicitly.